### PR TITLE
Made changes in coinChange.cpp according to style guide

### PR DESCRIPTION
**Pull request by mansiag**
**Fixes issue:**
N/A

**Changes:**
Made following changes in coinChange.cpp

- removed  using namespace std
- changed endl to \n
- added spaces between operators and operands

### CODE REVIEW

File: code/dynamic_programming/src/coin_change/coinchange.cpp [hunk 1]
```diff
@@ -3,23 +3,22 @@
 
 #include<iostream>
 #include<vector>
-using namespace std;
 
-const int MAX=100;
+const int MAX = 100;
 
-int coinWays(int amt,vector<int>& coins){
+int coinWays(int amt,std::vector<int>& coins){
```
Comment (iattempt) [inline code/dynamic_programming/src/coin_change/coinchange.cpp]: Add space after comma.
Add newline between close parenthesis and open brace.
And also remove blank line below.

File: code/dynamic_programming/src/coin_change/coinchange.cpp [hunk 2]
```diff
@@ -3,23 +3,22 @@
...
     // init the dp table
-    vector<int> dp(MAX, 0);
+    std::vector<int> dp(MAX, 0);
     int n = coins.size();
     dp[0] = 1; // base case
     
-    for(int j=0;j<n;j++){
-        for(int i=1;i<=amt;i++){
-            if(i-coins[j]>=0){
+    for(int j = 0; j < n; ++j){
```
Comment (iattempt) [inline code/dynamic_programming/src/coin_change/coinchange.cpp]: Add space before all open parenthesis of conditional statements.

File: code/dynamic_programming/src/coin_change/coinchange.cpp [hunk 3]
```diff
@@ -31,8 +30,8 @@ int coinWays(int amt,vector<int>& coins){
 
 int main(){
```
Comment (iattempt) [inline code/dynamic_programming/src/coin_change/coinchange.cpp]: Same here.

File: code/dynamic_programming/src/coin_change/coinchange.cpp [hunk 4]
```diff
@@ -6,30 +6,24 @@
...
+        for(int i = 1; i <= amt; ++i)
+            if(i - coins[j] >= 0)
                 // if coins[j] < i then add no. of ways - 
                 // - to form the amount by using coins[j]
-                dp[i] += dp[i - coins[j]]; 
-           }
-        }
-    }
-
+            dp[i] += dp[i - coins[j]];
```
Comment (iattempt) [inline code/dynamic_programming/src/coin_change/coinchange.cpp]: Missing indentation.

Comment (mansiag) [reply code/dynamic_programming/src/coin_change/coinchange.cpp]: Yeah Sorry

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**